src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -205,15 +208,30 @@ public ByteArrayOutputStream executeRequest(String urlString, ParamsMap params,
 	 * @return newly created httpClient
 	 */
 	private HttpClient createCustomHttpClient() {
-
+		
 		HttpClient httpClient = HttpClients.custom()
 				// set cookies validation on ignore
 				.setDefaultRequestConfig(RequestConfig.custom().setCookieSpec(CookieSpecs.IGNORE_COOKIES).build())
+				.setSSLSocketFactory(createSslConnectionSocketFactory())
 				.build();
 
 		return httpClient;
 	}
 
+	/**
+	 * Creates custom SSLConnectionSocketFactory and set it to use only the TLSv1.2 as supported protocol
+	 * 
+	 * @return newly created SSLConnectionSocketFactory
+	 */
+	private SSLConnectionSocketFactory createSslConnectionSocketFactory() {
+
+		SSLConnectionSocketFactory sslConnFactory = new
+				SSLConnectionSocketFactory(SSLContexts.createDefault(),
+				new String[] {"TLSv1.2"}, null,
```

Comment:
"TLSv1.2" can be extracted as a class-level constant. Also, please rename the rest of the constants, so all of them will use uppercase letters. Thank you!

Reply:
Done, thanks!